admin_guide/tcp_ingress_external_ports.adoc

```diff
@@ -0,0 +1,84 @@
+[[admin-guide-expose-external-ports]]
+= Enabling Ingress Traffic by Exposing External Ports
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+In {product-title} 3.2, you could use `*externalIP*` in the router service to control which external IP addresses expose the port. However, the OpenShift router could only listen on ports 80 and 443 for routing and mapping to backends (which have protocols). Previously, routes did not enable you to specify an external port. Port 80 runs in HTTP mode and 443 runs in TCP mode.
+
+However, you may want to expose external TCP traffic to the outside world.
+Starting in {product-title} 3.3, cluser administrators can expose an external
+port, enabling ingress traffic (TCP/UDP) routing to a definable backend.
+
+Exposing external ports allows you to allocate a unique IP address per service,
+rather than trying to manage the port space of a limited number of shared IP
+addresses. This ensures that each service can simply direct traffic from its
+chosen ports, regardless of the ports selected by other services.
+
+There are benefits to using `*IngressIP*` instead of `*externalIP*`:
+
+- An ingress IP is intended to be automatically assigned, and can only be assigned to one service. 
+- An external IP must be manually assigned, and conflicting assignments cannot be detected. Unfortunately, multiple services could potentially request the same IP, and the conflict would go undetected.
+
+== Restrictions
+
+To use `*IngressIP*`, you can either:
+
+- Select an IP address with the existing `*ExternalIP*` ingress.
++
+Or,
++
+- Have an IP address assigned from a pool. In this case, {product-title} implements a bare-metal version of the LoadBalancer service type and assigns IP addresses to the services. 
++ 
+[CAUTION]
+====
+You must ensure that the IP address pool you assign terminates at a node in your cluster. You can use the existing `*oadm ipfailover*` to configure IP failover for the external IPs.
+====
+
+Potential port clashes are handled on a first-come, first-served basis. If you
+request a port, it is only available if it has not yet been assigned. For
+example, `MongoDB service A` requests port 27017, and then `MongoDB service B`
+requests the same port; the first request gets the port, and the other receives
+a random port. This is similar to the route name allocation mechanism. 
+
+[NOTE]
+====
+You can only expose external ports in a sharded router environment, and only in
```

Comment:
This feature has nothing to do with the router, so the first requirement is incorrect.